src/main/webapp/assets/burgeremoji.svg

```diff
@@ -0,0 +1,24 @@
+<svg width="62" height="62" viewBox="0 0 62 62" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0)">
+<path d="M41.749 23.0547L41.3035 21.6169L16.3539 29.3488L16.7994 30.7865L18.1361 35.0997L44.5533 26.9131L43.2167 22.5999L41.749 23.0547Z" fill="#E74444"/>
```

Comment:
I love the SVG support of these emojis! They look super cute!

Reply:
thanks!